packages/restate-sdk-zod/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@restatedev/restate-sdk-zod",
+  "version": "1.4.0",
+  "description": "Restate Typescript SDK Zod",
+  "author": "Restate Developers",
+  "license": "MIT",
+  "email": "code@restate.dev",
+  "homepage": "https://github.com/restatedev/sdk-typescript#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/restatedev/sdk-typescript.git"
+  },
+  "bugs": {
+    "url": "https://github.com/restatedev/sdk-typescript/issues"
+  },
+  "type": "module",
+  "sideEffects": false,
+  "main": "./dist/cjs/src/public_api.js",
+  "types": "./dist/cjs/src/public_api.d.ts",
+  "module": "./dist/esm/src/public_api.js",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/esm/src/public_api.d.ts",
+        "default": "./dist/esm/src/public_api.js"
+      },
+      "require": {
+        "types": "./dist/cjs/src/public_api.d.ts",
+        "default": "./dist/cjs/src/public_api.js"
+      }
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "devDependencies": {
```

I think this should be
```suggestion
  "dependencies": {
```